apps/src/webapp/webapp.js

```diff
@@ -1328,6 +1345,30 @@ Webapp.createDropdown = function (opts) {
   return Boolean(divWebapp.appendChild(newSelect));
 };
 
+Webapp.getAttribute = function (opts) {
+  var divWebapp = document.getElementById('divWebapp');
+  var element = document.getElementById(opts.elementId);
+  var attribute = String(opts.attribute);
+  return divWebapp.contains(element) ? element[attribute] : false;
+};
+
+// Whitelist of HTML Element attributes which can be modified, to
+// prevent DOM manipulation which would violate the sandbox.
+Webapp.mutableAttributes = ['innerHTML', 'scrollTop'];
+
+Webapp.setAttribute = function (opts) {
+  var divWebapp = document.getElementById('divWebapp');
+  var element = document.getElementById(opts.elementId);
+  var attribute = String(opts.attribute);
+  var value = String(opts.value);
```

Comment:
independent of my comment above, should all attribute values be strings? is it possible that some should be numbers (like IMG width, height)? If you could just use opts.value directly, it might be better.